app/src/main/java/ch/hikemate/app/ui/saved/SavedHikesScreen.kt

```diff
@@ -184,6 +202,6 @@ private fun SavedHikesBottomMenu(
  * element of the enum will be the left-most section in the bottom menu.
  */
 enum class SavedHikesScreen(val label: String, @DrawableRes val icon: Int) {
-  Planned("Planned", R.drawable.calendar_today),
-  Saved("Saved", R.drawable.bookmark)
+  Saved("Saved", R.drawable.bookmark),
+  Planned("Planned", R.drawable.calendar_today)
```

Comment:
Question: Am I missing something here, or is it only a position change?

Reply:
It is only a position change to reflect the Figma.